src/main/java/co/refiere/resources/MetricsResource.java

```diff
@@ -38,6 +38,28 @@ public Response getCampaignsAmount(@PathParam("companyId") int companyId){
         response= String.format(response, referralsAmount);
         return Response.status(200).entity(response).build();
     }
+    
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/company/{companyId}/campaignsAmountNull")
```

Comment:
Change the route, instead of `campaignAmountNull` use something more descriptive like: `referresAmount`